robot_skills/src/robot_skills/amigo.py

```diff
@@ -1,9 +1,33 @@
 #! /usr/bin/env python
 
 import robot
+import rospy
+import tf
+import math
+
+from geometry_msgs.msg import PoseStamped
+
+
+class SSL(object):
```

Comment:
Something went wrong with merging because there are a lot of `robot_smach_states` commits here. I'll help you next Tuesday with this.